Makefile

```diff
@@ -0,0 +1,79 @@
+PROJECT_NAME := andelasocietiesbackend
+REPO_NAME ?= andela-societies-backend
+
+# File names
+DOCKER_TEST_COMPOSE_FILE := docker/test/docker-compose.yml
+
+# Docker compose project names
+DOCKER_TEST_PROJECT := "$(PROJECT_NAME)test"
+
+.PHONY: help
+
+
+## Show help
+help:
+	@echo ''
+	@echo 'Usage:'
+	@echo '${YELLOW} make ${RESET} ${GREEN}<target> [options]${RESET}'
+	@echo ''
+	@echo 'Targets:'
+	@awk '/^[a-zA-Z\-\_0-9]+:/ { \
+    	message = match(lastLine, /^## (.*)/); \
+		if (message) { \
+			command = substr($$1, 0, index($$1, ":")-1); \
+			message = substr(lastLine, RSTART + 3, RLENGTH); \
+			printf "  ${YELLOW}%-$(TARGET_MAX_CHAR_NUM)s${RESET} %s\n", command, message; \
+		} \
+	} \
+    { lastLine = $$0 }' $(MAKEFILE_LIST)
+	@echo ''
+    
+## Generate Virtual environment
+venv:
```

Comment:
@nyambati I don't think all team members are on board with that, I'm the one who introduced that pipenv mainly because I experiment a lot on my local setup and most of the time I end up polluting the requirement.txt with unnecessary packages pipenv seems to cater to such a problem. The consensus for now is as is, using the normal venv without pipenv.

Reply:
@nyambati removed this for now as discussed.